splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/joins/CrossJoinIT.java

```diff
@@ -713,5 +713,42 @@ public void testCharTimestampColumnsCrossJoin() throws Exception {
         rs.close();
     }
 
+    @Test
+    public void testCrossJoinPreserveSortOrderNoJoinStrategyHint() throws Exception {
+        String sqlText = format("select tt2.a from \n" +
+                "tab2 tt2 --splice-properties useSpark=%s\n" +
+                "inner join %s ttb\n" +
+                "on ttb.c2 < 5 and tt2.a < 4 order by tt2.a", useSpark, bigTable);
+
+        if (useSpark) {
+            ResultSet rs = classWatcher.executeQuery("explain " + sqlText);
```

Comment:
use try-with-resources here to auto close the resultset

Reply:
Done.